.editorconfig

```diff
@@ -1,8 +1,92 @@
-; top-most EditorConfig file
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
 root = true
 
-; Unix-style newlines
 [*]
-end_of_line = LF
+# Change these settings to your own preference
+indent_style = space
+indent_size = 4
+
+# We recommend you to keep these unchanged
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.feature]
+indent_style = space
+indent_size = 2
+
+[*.js]
+indent_style = space
+indent_size = 4
+
+[*.json]
+indent_style = space
+indent_size = 4
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.php]
+indent_style = space
+indent_size = 4
+
+[*.sh]
+indent_style = tab
+indent_size = 4
+
+[*.vcl]
```

Comment:
remove, because no varnish configuration in the `bolt/core` repository.

Reply:
Ok, will do.

Reply:
I just corrected that in this [pull request](https://github.com/bolt/project/pull/47/files)